plugins/lsp/default.nix

```
@@ -192,12 +192,13 @@ in
       keymapsOnEvents.LspAttach =
         let
           mkMaps =
-            prefix:
+            prefix: descPrefix:
             mapAttrsToList (
               key: action:
               let
                 actionStr = if isString action then action else action.action;
-                actionProps = if isString action then { } else filterAttrs (n: v: n != "action") action;
+                actionProps = optionalAttrs (!(isString action)) (filterAttrs (n: v: n != "action") action);
+                desc = descPrefix + " " + actionStr;
```

```suggestion
                desc = "${descPrefix} ${actionStr}";
```
I guess this looks simpler...